pymeasure/instruments/thorlabs/thorlabs_elliptec.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Created the 15/06/2023
+
+@author: Sebastien Weber
+"""
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.thorlabs.elliptec_utils.tools import parse
+from pymeasure.instruments.thorlabs.elliptec_utils.base import MotorChannel, ElliptecController
+
+import logging
+
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class RotatorChannel(MotorChannel):
```

Comment:
Why are these channels in the main file, but the MotorChannel, and even more, the ElliptecController itself not?